QuizApp/Common/Helpers/String+Extensions.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension String {
+
+    static let appName = "PopQuiz"
+
+}
```

Comment:
If you plan to have a file with strings commonly used in the app (in the future event localisations), I'd suggest creating a struct or an enum rather than extending string.